bundle/rhpam/manifests/rhpam-kogito-operator-manager-config_v1_configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+    kind: ControllerManagerConfig
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: d1731e98.kiegroup.org
```

Comment:
is this random name correct?

Reply:
this value is same as Leader election id https://github.com/vaibhavjainwiz/kogito-operator/blob/761325423fb93811a561c18027976f2ea8aca22a/main.go#L75

maintained by operator-sdk itself